s2i/README.md

```diff
@@ -0,0 +1,66 @@
+# Source-to-image (a.k.a. `s2i`)
+
+This build tempalte builds source into a container image using
+Openshift [`source-to-image`](https://github.com/openshift/source-to-image)
+build tool.
+
+> Source-to-Image (S2I) is a toolkit and workflow for building
+> reproducible Docker images from source code. S2I produces
+> ready-to-run images by injecting source code into a Docker container
+> and letting the container prepare that source code for execution. By
+> creating self-assembling builder images, you can version and control
+> your build environments exactly like you use Docker images to
+> version your runtime environments.
+
+This build template uses `s2i` support for generating a `Dockerfile`
+and a build context, that can be later build by [`buildah`](https://github.com/containers/buildah).
+This allows user to have, *somewhat*, similar builds as Openshift
+[`Source-to-Image` build stragety](https://docs.openshift.com/container-platform/3.11/architecture/core_concepts/builds_and_image_streams.html#source-build)
+
+## Parameters
+
+* **BUILDER_IMAGE** : The name of the image containing the `s2i` tool
+  (_default:_: vdemeester/s2i-builder).
+* **BUILDAH_BUILDER_IMAGE** : The name of the image containing the
+  `buildah` tool (_default:_ vdemeester/buildah-builder)
```

Comment:
`docker.io/vdemeester/buildah-builder`? `gcr.io/vdemeester/buildah-builder`? `quay.io/vdemeester/buildah-builder`? all of them? none of them? :wink: